src/chrome/content/rules/Codeblocks.org.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+	This domain contains a wildcard DNS record.
+
+	Invalid certificate:
+		www.codeblocks.org
+
+	Mismatch:
+		forums.codeblocks.org
+		wiki.codeblocks.org
```

Comment:
these 2 work for me, would you please check again ?

Reply:
Both subdomains show on https an other content than on http. On https both subdomains show the content of the frontpage as shown on https://codeblocks.org.

Reply:
I see, can you mark them as `Different content`?